app/styles/components/homepage-image-grid.css

```diff
@@ -45,6 +45,10 @@
   width: max-content;
 }
 
+[class*=well-] img {
+  width: auto;
```

Comment:
This was needed as the styles for images in wells (coming from ember-styleguide) would cause images with defined `width` and `height` attributes (which ember-responsive-images sets automatically and is a [recommended practice again](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)) to have distorted dimensions (ignoring the explicit aspect ratio given by these attributes) in Safari and FF (but not Chrome!). 

The easiest fix for this was to add this here, but I can also try to upstream this (which should not have any adverse effects when the attributes are not set) to ember-styleguide!?